src/lib/commands/remoteFs.ts

```diff
@@ -0,0 +1,28 @@
+import { ExtensionContext, window, commands } from 'vscode';
+import { COMMANDS } from '../constants';
+const { deleteFile } = require('@hubspot/cli-lib/api/fileMapper');
+const { getPortalId } = require('@hubspot/cli-lib');
+
+export const registerCommands = (context: ExtensionContext) => {
+  context.subscriptions.push(
+    commands.registerCommand(
+      COMMANDS.REMOTE_FS.DELETE,
+      async (clickedFileLink) => {
+        console.log(COMMANDS.REMOTE_FS.DELETE);
+        const fileLinkIsFolder = clickedFileLink.icon === 'symbol-folder';
+        const filePath = fileLinkIsFolder
+          ? clickedFileLink.path
+          : clickedFileLink.url.slice(5);
```

Comment:
Slice here because if it's not a folder its url will have the prefix "hubl:" to make it openable by the `remoteFileProvider`